src/Util/ToggleLadderByLadderItem.cs

```diff
@@ -27,7 +27,12 @@ public class ToggleLadderByLadderItem : MonoBehaviour {
             }
 
             foreach (GameObject gameObject in optionalObjectsToDisable) {
-                gameObject.SetActive(ladderItem.Quantity > 0);
+                if(ladderItem.Quantity == 0 && gameObject.name.Contains("Stone Switch")) {
+                    //instead of removing the switches (causing a ton of lag when an every-frame coroutine tries to run on them), lets just move them really far away
+                    gameObject.transform.position = new Vector3(10000, 10000, 10000);
+                } else {
+                    gameObject.SetActive(ladderItem.Quantity > 0);
+                }
```

Comment:
```suggestion
                if (ladderItem.Quantity == 0 && gameObject.name.Contains("Stone Switch")) {
                    //instead of removing the switches (causing a ton of lag when an every-frame coroutine tries to run on them), lets just move them really far away
                    gameObject.transform.position = new Vector3(10000, 10000, 10000);
                } else {
                    gameObject.SetActive(ladderItem.Quantity > 0);
                    if (gameObject.name == "Stone Switch") {
                        gameObject.transform.position = new Vector3(18, 52, -64);
                    }
                    if (gameObject.name == "Stone Switch (1)") {
                        gameObject.transform.position = new Vector3(16, 52, -26);
                    }
                }
```